src/main/drivers/at32/serial_uart_at32f43x.c

```diff
@@ -230,6 +242,72 @@ const uartHardware_t uartHardware[UARTDEV_COUNT] = {
     },
 #endif
 
+#ifdef USE_UART6
+    {
+        .device = UARTDEV_6,
+        .reg = USART6,
+#ifdef USE_DMA
+        .rxDMAMuxId = DMAMUX_DMAREQ_ID_USART6_RX,
+        .rxDMAResource = (dmaResource_t *)UART6_RX_DMA_CHANNEL,
+        .txDMAMuxId = DMAMUX_DMAREQ_ID_USART6_RX,
+        .txDMAResource = (dmaResource_t *)UART6_TX_DMA_CHANNEL,
+#endif
+        .rxPins = {
+            { DEFIO_TAG_E(PA5),  GPIO_MUX_8 },
+            { DEFIO_TAG_E(PA12), GPIO_MUX_8 },
+            { DEFIO_TAG_E(PC7),  GPIO_MUX_8 },
+            { DEFIO_TAG_E(PG9), GPIO_MUX_8 },
```

Comment:
Indentation. Either fix them all or none.
```suggestion
            { DEFIO_TAG_E(PG9),  GPIO_MUX_8 },
```